File: medium/Q5/arraytoset.js
// Function to convert an array to set
const arrayToSet = (arr) => new Set(arr);

console.log(
  JSON.stringify([...arrayToSet([1, 2, 3, 4, 5, 2, 4, 1, 3, 2, 5])]) ===
    JSON.stringify([1, 2, 3, 4, 5])
); //true

File: medium/Q5/arraytoset.py
def array_to_set(arr):
    """ Function to convert an array to set """
    return set(arr)

if __name__ == "__main__":
    print(array_to_set([1, 2, 3, 4, 5, 2, 4, 1, 3, 2, 5]) == {1, 2, 3, 4, 5}) # True